static/app/components/forms/fields/booleanField.tsx

```diff
@@ -59,12 +59,12 @@ export default class BooleanField extends Component<BooleanFieldProps> {
           const handleChange = this.handleChange.bind(this, value, onChange, onBlur);
 
           const {type: _, ...propsWithoutType} = props;
-          const switchProps = {
+          const switchProps: SwitchProps = {
             ...propsWithoutType,
-            size: 'lg' as React.ComponentProps<typeof Switch>['size'],
-            isActive: !!value,
-            isDisabled: disabled,
-            toggle: handleChange,
+            size: 'lg' satisfies SwitchProps['size'],
```

Comment:
do you need this with :SwitchProps you added above
```suggestion
            size: 'lg'
```